client/state/themes/selectors/can-use-theme.js

```diff
@@ -39,7 +40,11 @@ export function canUseTheme( state, siteId, themeId ) {
 	}
 
 	if ( type === DOT_ORG_THEME ) {
-		return siteHasFeature( state, siteId, FEATURE_INSTALL_THEMES );
+		return (
+			siteHasFeature( state, siteId, FEATURE_INSTALL_THEMES ) ||
+			// Atomic sites are tested above through the features system
+			isJetpackSite( state, siteId, { treatAtomicAsJetpackSite: false } )
```

Comment:
@mmtr unsure if this is correct, or if `wpcom_features` needs changing. It doesn't have an entry for the INSTALL_THEMES feature (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sjcpbz%2Qsrngherf%2Spynff%2Qjcpbz%2Qsrngherf.cuc%3Se%3Q228736oq%23750-og)

Reply:
I think we added `JETPACK_ALL_SITES` for this specific reason (fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Szh%2Qcyhtvaf%2Sjcpbz%2Qsrngherf%2Spynff%2Qjcpbz%2Qsrngherf.cuc%3Se%3Q228736oq%231491-og), so can we maybe update `INSTALL_THEMES` with `JETPACK_ALL_SITES`?

Reply:
Cheers, opened D161413-code instead